polyfill/test/usercalendar.mjs

```diff
@@ -16,60 +16,60 @@ import * as Temporal from 'proposal-temporal';
 
 describe('Userland calendar', () => {
   describe('Trivial subclass', () => {
-    // For the purposes of testing, a nonsensical calendar that uses 0-based
+    // For the purposes of testing, a nonsensical calendar that uses 2-based
     // month numbers, like legacy Date
```

Comment:
I'm very sad that this is no longer possible :crying_cat_face: 

In any case, "like legacy Date" no longer applies.
```suggestion
    // month numbers, instead of 1-based
```